src/cdpTargetsProvider.ts

```diff
@@ -139,6 +140,11 @@ export class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTarget> {
                             actualTarget.faviconUrl = filePath;
                             resolve(actualTarget);
                         } catch (e) {
+                            void ErrorReporter.showErrorDialog({
```

Comment:
I don't think this is worth showing an error to the user for. It does not significantly affect workflow, and if it fails on a page with several targets, could get really irritating